modules/riemann/riemann-grammar.ym

```diff
@@ -182,14 +182,14 @@ riemann_tls_options
         ;
 
 riemann_tls_option
-        : KW_CA_FILE '(' string ')' { riemann_dd_set_tls_cacert(last_driver, $3); free($3); }
-        | KW_CERT_FILE '(' string ')' { riemann_dd_set_tls_cert(last_driver, $3); free($3); }
-        | KW_KEY_FILE '(' string ')' { riemann_dd_set_tls_key(last_driver, $3); free($3); }
+        : KW_CA_FILE '(' path_secret ')' { riemann_dd_set_tls_cacert(last_driver, $3); free($3); }
+        | KW_CERT_FILE '(' path_secret ')' { riemann_dd_set_tls_cert(last_driver, $3); free($3); }
```

Comment:
What is the use case of `path_secret`? I think the only secret is the key. Cert and CA can be made to public. They might be useful in the debug bundle, while debugging tls connectivity related problems.

Reply:
- path_secret is used for key and key like material.
- You are correct for Cert and CA that it can be made to public. They will be useful in the debun script. I will remove ```path_secret``` from them.